qiskit/transpiler/preset_passmanagers/level1.py

```diff
@@ -129,6 +129,6 @@ def _opt_control(property_set):
         # pm1.append(_direction_check)  # TODO
         pm1.append(_direction, condition=_direction_condition)
     pm1.append(_reset)
-    pm1.append(_depth_check + _opt, do_while=_opt_control)
+    pm1.append(_opt + _depth_check, do_while=_opt_control)
```

Comment:
i intentionally put `depth_check` first because if the `opt` is not successful in reducing depth, `opt` should not run a second time.
if `opt` is first, it will always run at least twice.

Reply:
`_opt_control` makes a decition on a property might not be updated. I can move this change somewhere else I we can discuss it later.